crypto/bls/attestation_test.go

```diff
@@ -0,0 +1,41 @@
+package bls
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBlsKeyCreation(t *testing.T) {
+	var tests = map[string]struct {
+		keyPath  string
+		password string
+		wantErr  bool
+	}{
+		"valid bls key save": {
+			keyPath:  "./operator_keys/test.bls.key.json",
```

Comment:
is ./operator_keys a path used for anything other than this test? If not this will delete potential user data.
Maybe rename to `operator_keys_test_dir` or something?

Reply:
oh ya can rename.